### PR TITLE
[Limit Orders]: Add order type selector

### PR DESCRIPTION
**Pull request by fabryscript**
## What is the purpose of the change:

Implement Order type dropdown menu

### Linear Task

[Linear Task URL](https://linear.app/osmosis/issue/LIM-115/[limitmarket]-order-type-selection)

### CODE REVIEW

File: packages/web/components/trade-tool/index.tsx [hunk 1]
```diff
@@ -23,7 +24,10 @@ export const TradeTool: FunctionComponent<TradeToolProps> = () => {
   return (
     <ClientOnly>
       <div className="relative flex flex-col gap-6 overflow-hidden md:gap-6 md:px-3 md:pb-4 md:pt-4">
-        <SwapToolTabs activeTab={tab} setTab={setTab} />
+        <div className="flex w-full items-center justify-between">
+          <SwapToolTabs activeTab={tab} setTab={setTab} />
+          <OrderTypeSelector />
```
Comment (crnbarr93) [inline packages/web/components/trade-tool/index.tsx:29]:
```suggestion
          {tab !== SwapToolTab.SWAP && <OrderTypeSelector />}
```

Should be disabled for the Swap tab

File: packages/web/components/swap-tool/order-type-selector.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+import classNames from "classnames";
+import { parseAsStringLiteral, useQueryState } from "nuqs";
+import React, { Fragment, useMemo } from "react";
+
+import { Icon } from "~/components/assets";
+import { SpriteIconId } from "~/config";
+import { useTranslation } from "~/hooks";
+
+interface UITradeType {
+  id: "market" | "limit" | "recurring";
```
Comment (crnbarr93) [inline packages/web/components/swap-tool/order-type-selector.tsx:11]: I think we can leave out "recurring" for now.

**Review by crnbarr93 — APPROVED** [review]

File: packages/web/components/trade-tool/index.tsx (hunk 1 above)
Comment (fabryscript) [reply packages/web/components/trade-tool/index.tsx]: [aa9f5cb](https://github.com/osmosis-labs/osmosis-frontend/pull/3320/commits/aa9f5cbb3d63fc339174e34f1a5487f93855a1d6)

File: packages/web/components/swap-tool/order-type-selector.tsx (hunk 2 above)
Comment (fabryscript) [reply packages/web/components/swap-tool/order-type-selector.tsx]: [7292db4](https://github.com/osmosis-labs/osmosis-frontend/pull/3320/commits/7292db403df0dde7aa6409b806c06ca7bce4c92c)

**Pull request merged**